bucket/winssh-pageant.json

```diff
@@ -0,0 +1,43 @@
+{
+    "version": "1.2",
+    "description": "Pageant to Windows OpenSSH agent bridge",
+    "homepage": "https://github.com/ndbeals/winssh-pageant",
+    "license": "BSD-3-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ndbeals/winssh-pageant/releases/download/v1.2/winssh-pageant-v1.2_amd64.zip",
+            "hash": "a9bef9d134e872e60fd1d0950ce29f73b98af4802c9e93987a4ecafe8debf814"
+        },
+        "32bit": {
+            "url": "https://github.com/ndbeals/winssh-pageant/releases/download/v1.2/winssh-pageant-v1.2_386.zip",
+            "hash": "0591f0c4475dba8fbfacc7ffe5fcb77ac384008d884976b386b4f25fa894e3a8"
+        }
+    },
+    "extract_dir": "",
+    "bin": "winssh-pageant.exe",
+    "shortcuts": [
+        [
+            "winssh-pageant.exe",
+            "..\\Startup\\winssh-pageant"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/ndbeals/winssh-pageant/releases/download/v$version/winssh-pageant-v$version_amd64.zip",
+                "hash": {
+                    "url": "https://github.com/ndbeals/winssh-pageant/releases/tag/v$version/",
+                    "regex": "amd64:$sha256"
+                }
+            },
+            "32bit": {
+                "url": "https://github.com/ndbeals/winssh-pageant/releases/download/v$version/winssh-pageant-v$version_386.zip",
+                "hash": {
+                    "url": "https://github.com/ndbeals/winssh-pageant/releases/tag/v$version/",
+                    "regex": "386:$sha256"
```

Comment:
Use parsing of the table rather than some text, which will be most likely removed in future

Reply:
Do you refer to the table of hashes on the release description?

If so, that table are hashes of the executable themselves, because the go binary is reproducible, so users may check it themselves. 

I can add another table of the zip file hashes to the release description, but wouldn't it be using regex to match it anyways? I guess I'm confused what you mean by parse the table?

Reply:
I'm retrieving the hash with regex just like [anaconda3](https://github.com/lukesampson/scoop-extras/blob/master/bucket/anaconda3.json) and [accesser](https://github.com/lukesampson/scoop-extras/blob/master/bucket/accesser.json) are. I can remove the hash checking if you'd like.